packages/core/nuxt-module/lib/plugins/cypress.js

```diff
@@ -0,0 +1,13 @@
+import Vue from 'vue';
```

Comment:
**suggestion** perhaps its better to  keep this plugin in the project as its a part of the theme and is usually project-specific